src/tutorials/race_timer.md

```diff
@@ -0,0 +1,4385 @@
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/RaceTimerTutorial/cc_template_deinstance.mp4" type="video/mp4" />
+    </video>
+</div>
+
+If you enter **Play** mode you will notice the spawn position isn't in an ideal location for any player that joins the game. Move the **Spawn Point** so that it's before the starting position on the track.
+
+!!! tip
+    Pressing ++V++ will toggle the visibility of Gizmos. With gizmo visibility on, it's easy to see where to move objects such as the **Spawn Point**.
+
+![!Spawn Point](../img/RaceTimerTutorial/move_spawn_point.png){: .center loading="lazy" }
+
+## Starting Line
+
+When a race is about to start, we need to move the players to the starting line. Instead of grouping all players up in one spot, we can create a few positions along the starting line and randomly pick one to place the player at. There is still a possibility of players spawning in the same location, and to fix this it would require a more advanced system to keep track of which lanes have been assigned. For this tutorial we will keep it simple.
+
+### Creating Lane Positions
+
+!!! tip
+    As a visual aid to help move the starting positions, an object such as a **Cube** can be used. For performance it's recommended to use a **Group** when needing to reference a position for spawning in objects. **Group** objects have minimal properties (i.e. no **Rendering**) compared to other objects.
+
+1. Create a new group called **Starting Positions** inside the **Race Timer Tutorial - Track** group.
+2. Drag and drop a **Cube** from **Core Content** into the **Starting Positions** group.
+3. Set **Game Collision** to **Force Off** for the **Cube** object.
+4. Set **Camera Collision** to **Force Off** for the **Cube** object.
+5. Rename **Cube** to **Position**.
+6. Duplicate and space these out along the starting line.
+7. Set **Visibility** to **Force Off** for all **Position** objects.
+
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/RaceTimerTutorial/create_start_positions.mp4" type="video/mp4" />
+    </video>
+</div>
+
+## Setup Script Hierarchy
+
+We need to setup the script folder and contexts in the **Hierarchy**.
+
+1. Create a folder called **Scripts** inside the **Race Timer Tutorial - Track** group.
+2. Create a **Client Context** inside the **Scripts** folder.
+3. Create a **Server Context** inside the **Scripts** folder.
+
+### Client Context vs Server Context
+
+Understanding the difference between these two contexts can help you understand where to place **Lua** scripts and what they should be used for.
+
+Anything in the **Client Context** exists on the clients version of the game. This means that there is no network traffic, and anything that gets modified is applied to that local player and not all players on the server.
```

Comment:
* on each player's version of the game.

Also, just a side note here, I keep finding client context things that are surprisingly silently replicated anyway. Is this just me? IK anchors, apparently client-side trigger interaction events (only).

Reply:
Hmm, that's interesting about client things being replicated. Maybe I should change how I phrase it so say "most things"?

"Most things in the **Client Context** exists on each player's version of the game."